cogs/code_utils/_ref.py

```diff
@@ -192,9 +199,16 @@ async def haskell_ref(ctx, text):
             soup = BeautifulSoup(await response.text(), 'lxml').find('div', id='content')
 
             title = soup.find('h1', id='firstHeading').string
-            description = '\n'.join([markdownify(p) for p in  soup.find_all(lambda x: x.name in ['p', 'li'] and tuple(x.parents)[1].name not in ('td', 'li'), limit=6)])[:2048]
+            description = '\n'.join(
+                [
+                    markdownify(p) for p in soup.find_all(
+                        lambda x: x.name in ['p', 'li'] and
+                        tuple(x.parents)[1].name not in ('td', 'li'), limit=6
+                    )
+                ]
+            )[:2048]
```

Comment:
```suggestion
            description = '\n'.join([
                markdownify(p) for p in soup.find_all(
                    lambda x: x.name in ['p', 'li'] and
                    tuple(x.parents)[1].name not in ('td', 'li'), limit=6
                )
            ])[:2048]
```
This is a bit easier on eyes